### ISSUE TITLE
Updating props does not re-render map

### ISSUE THREAD

**Issue by altoning** [kadoshms/react-jvectormap#13]
Updating the props (eg. backgroundColor) does not re-render the map. How can we get this to work?

```
          <VectorMap
            map={'world_mill'}
            backgroundColor={ colors[0] }
            containerStyle={{
              width: "99%",
              height: "100%",
              paddingLeft: "0.5%",
              paddingRight: "0.5%",
            }}
            series={{
              regions: [
                {
                  values: myData,
                  scale: colors,
                  legend: {vertical: false},
                }
              ]
            }}
          />
```

**Comment by kadoshms** [comment]
fixed b331b6ceee8a731cde4e3fc9dfe810f4b276d866

**Comment by Lakshitnagar** [comment]
If you don't want to work on forked stuff, here is a work around.

First create a boolean state to how/hide the map:

    const [isShowMap, setIsShowMap] = useState(true);

Attached this state to show/hide the map html:

    { isShowMap && <VectorMap... }

On any change in props or any data, just turn this flag off and on again with setTimeout:

    useEffect(()=>{
      setIsShowMap(false);
      setTimeout(()=>{
        setIsShowMap(true);
      })
    }, [prop1, prop2]);

As it is just a work around, you will experience a tittle flicker.
